sklearn/neighbors/regression.py

```diff
@@ -292,15 +296,29 @@ def predict(self, X):
         _y = self._y
         if _y.ndim == 1:
             _y = _y.reshape((-1, 1))
-
+            
+        empty_obs = np.full_like(_y[0], np.nan)
+        
         if weights is None:
-            y_pred = np.array([np.mean(_y[ind, :], axis=0)
-                               for ind in neigh_ind])
+            y_pred = np.array([np.mean(_y[ind, :],
+                                       axis=0)
```

Comment:
please consider to fill the current line before starting a new line, other places the same.